widget/scroller.go

```diff
@@ -159,14 +244,16 @@ var _ desktop.Hoverable = (*scrollBarArea)(nil)
 type scrollBarArea struct {
 	BaseWidget
 
-	isWide bool
-	scroll *ScrollContainer
+	isWide      bool
+	isTall      bool
+	scroll      *ScrollContainer
+	orientation scrollBarOrientation
 }
 
 func (s *scrollBarArea) CreateRenderer() fyne.WidgetRenderer {
 	s.ExtendBaseWidget(s)
-	bar := newScrollBar(s)
-	return &scrollBarAreaRenderer{area: s, bar: bar, objects: []fyne.CanvasObject{bar}}
+	bar := newScrollBar(s, s.orientation)
```

Comment:
Here you are passing `s` and then `s.orientation`.
Would it not be better to just reference the area's orientation through the area itself rather than caching a copy of this variable?
Essentially newScrollBar does not need the extra parameter - the first one contains all the info you need to pass